src/modules/user/model/wallet.entity.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+  OneToOne,
+  JoinColumn,
+  Unique,
+} from "typeorm";
+import { User } from "./user.entity";
+
+@Entity()
+@Unique("UK_wallet_id_walletAddress", ["id", "walletAddress"])
+export class Wallet {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ unique: true })
```

Comment:
Can we use the same approach as we used for the `UK_wallet_id_walletAddress` constraint and set it as a decorator attached to the `Wallet` class? It's nice to have a single place to look for indexes and constraints

Reply:
done